MyCoRe-Org/mycore
# MCR-3092 XSL test capabilities

[Link to jira](https://mycore.atlassian.net/browse/MCR-3092).

MCRTestCase contains new methods
* parse: parses a given XML structure using a given XSL file
* xmlTest: encapsulates an XML structure in an XML document for test

Usage:
* create a test.xsl file containing match templates calling functions/templates to test (e.g. classificationTest.xsl)
* parse a structure/empty XML using the parse-method in MCRTestCase
* call Junit-assert-statements using the resulting XML structure

see usage example in XSLClassificationTest
